### PR TITLE
Add node IP for advertised.listeners

### PR DESCRIPTION
**Pull request by bartam1**
| Q               | A |
| --------------- | --- |
| Bug fix?        | no |
| New feature?    | yes |
| API breaks?     | no |
| Deprecations?   | no |
| License         | Apache 2.0 |

### What's in this PR?

Add doc about node IP for advertise.listeners

### CODE REVIEW

File: docs/external-listener/index.md [hunk 1]
```diff
@@ -144,6 +144,32 @@ To configure an external listener that uses the NodePort access method, complete
 
 ### NodePort external IP
 
+The node IP of the node where the broker pod is scheduled will be used in the advertized.listeners broker configuration when the `nodePortNodeAddressType` is specified.
```
Comment (panyuenlau) [inline docs/external-listener/index.md:147]:
```suggestion
The node IP of the node where the broker pod is scheduled will be used in the advertised.listeners broker configuration when the `nodePortNodeAddressType` is specified.  
```

File: docs/external-listener/index.md [hunk 2]
```diff
@@ -144,6 +144,32 @@ To configure an external listener that uses the NodePort access method, complete
 
 ### NodePort external IP
 
+The node IP of the node where the broker pod is scheduled will be used in the advertized.listeners broker configuration when the `nodePortNodeAddressType` is specified.  
+Its value determine which IP or domain name of the Kubernetes node will be used. It can be: Hostname, ExternalIP, InternalIP, InternalDNS and ExternalDNS.  
+The hostNameOverride and nodePortExternalIP must be not specified in this case.
```
Comment (panyuenlau) [inline docs/external-listener/index.md:149]:
```suggestion
The hostNameOverride and nodePortExternalIP must not be specified in this case.
```

File: docs/external-listener/index.md [hunk 3]
```diff
@@ -144,6 +144,32 @@ To configure an external listener that uses the NodePort access method, complete
 
 ### NodePort external IP
 
+The node IP of the node where the broker pod is scheduled will be used in the advertized.listeners broker configuration when the `nodePortNodeAddressType` is specified.  
+Its value determine which IP or domain name of the Kubernetes node will be used. It can be: Hostname, ExternalIP, InternalIP, InternalDNS and ExternalDNS.
```
Comment (panyuenlau) [inline docs/external-listener/index.md:148]:
```suggestion
Its value determines which IP or domain name of the Kubernetes node will be used, the possible values are: Hostname, ExternalIP, InternalIP, InternalDNS and ExternalDNS.  
```

**Review by panyuenlau — COMMENTED** [review]
Thanks for updating this, just some grammatical nits

**Review by pregnor — COMMENTED** [review]
LGTM with Darren's comments

**Review by fekete-robert — DISMISSED** [review]

File: docs/external-listener/index.md [hunk 4]
```diff
@@ -144,6 +144,32 @@ To configure an external listener that uses the NodePort access method, complete
 
 ### NodePort external IP
 
+The node IP of the node where the broker pod is scheduled will be used in the advertise.listeners broker configuration when the `nodePortNodeAddressType` is specified.
```
Comment (Kuvesz) [inline docs/external-listener/index.md:147]:
```suggestion
The node IP of the node where the broker pod is scheduled will be used in the advertised.listeners broker configuration when the `nodePortNodeAddressType` is specified.  
```

**Review by Kuvesz — COMMENTED** [review]
Found a small typo

**Review by Kuvesz — DISMISSED** [review]

**Review by panyuenlau — APPROVED** [review]

**Review by pregnor — DISMISSED** [review]
LGTM

**Comment by pregnor** [comment]
Merging because Koperator [v0.24.0](https://github.com/banzaicloud/koperator/releases/tag/v0.24.0) has been released.

**Comment by pregnor** [comment]
There were no changes in my force-pushes, I merely rebased onto latest master.

**Review by pregnor — APPROVED** [review]

**Pull request merged**